### PR TITLE
feat(routes): Add censorbib

### PR DESCRIPTION
**Pull request by xtexChooser**
## Example for the Proposed Route(s) / 路由地址示例

```routes
/nymity/censorbib
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

https://github.com/NullHypothesis/censorbib/issues/8

### CODE REVIEW

File: lib/routes/censorbib/censorbib.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    ],
+    name: 'CensorBib Updates',
+    maintainers: ['xtexChooser'],
+    handler,
+    url: 'censorbib.nymity.ch/',
+    description: `CensorBib `,
+};
+
+async function handler() {
+    const resp = await got.get(url, {});
```
Comment (TonyRL) [inline lib/routes/censorbib/censorbib.ts:32]:
```suggestion
    const resp = await got.get(url);
```

File: lib/routes/censorbib/censorbib.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    example: '/censorbib',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
```
Comment (TonyRL) [inline lib/routes/censorbib/censorbib.ts:18]:
```suggestion
```

File: lib/routes/censorbib/censorbib.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    radar: [
+        {
+            source: ['censorbib.nymity.ch/'],
+        },
+    ],
+    name: 'CensorBib Updates',
+    maintainers: ['xtexChooser'],
+    handler,
+    url: 'censorbib.nymity.ch/',
+    description: `CensorBib `,
```
Comment (TonyRL) [inline lib/routes/censorbib/censorbib.ts:28]:
```suggestion
```

File: lib/routes/censorbib/censorbib.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            const id = c.attr('id')!;
+            const title = c.find('span.paper').text().trim();
+            const author = c.find('span.author').text().trim();
+            const other = c.find('span.other').text().trim();
+            const download = c.find("img.icon[title='Download paper']").parent().attr('href');
+            const downloadBibTex = c.find("img.icon[title='Download BibTeX']").parent().attr('href');
+            const linkToPaper = c.find("img.icon[title='Link to paper']").parent().attr('href');
+            return {
+                title,
+                description: `${other}\n\n<a href='${download}'>Download</a>\n<a href='${downloadBibTex}'>Download BibTex</a>`,
```
Comment (TonyRL) [inline lib/routes/censorbib/censorbib.ts:48]: Please use `<br>` for linebreak.

File: lib/routes/censorbib/namespace.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
```
Comment (TonyRL) [inline lib/routes/censorbib/namespace.ts:1]: Please move the [namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) to `nymity` instead.

**Comment by xtexChooser** [comment]
wtf happened to the Semgrep CI

File: lib/routes/nymity/censorbib.ts [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+            const id = c.attr('id')!;
+            const title = c.find('span.paper').text().trim();
+            const author = c.find('span.author').text().trim();
+            const other = c.find('span.other').text().trim();
+            const download = c.find("img.icon[title='Download paper']").parent().attr('href');
+            const downloadBibTex = c.find("img.icon[title='Download BibTeX']").parent().attr('href');
+            const linkToPaper = c.find("img.icon[title='Link to paper']").parent().attr('href');
+            return {
+                title,
+                description: `${other}<br/><br/><a href='${download}'>Download</a>\n<a href='${downloadBibTex}'>Download BibTex</a>`,
```
Comment (TonyRL) [inline lib/routes/nymity/censorbib.ts:38]:
```suggestion
                description: `${other}<br/><br/><a href='${download}'>Download</a><br/><a href='${downloadBibTex}'>Download BibTex</a>`,
```

**Pull request merged**